### PR TITLE
Update with-universal-configuration example

### LINKED ISSUE #4498: Update with-universal-configuration example
# Feature request

## Is your feature request related to a problem? Please describe.
It is confusing for new users to get environment variables set up because there are examples that do not work, and there is an officially supported way of doing environment variables now

https://github.com/zeit/next.js#exposing-configuration-to-the-server--client-side

## Describe the solution you'd like
Remove `with-universal-configuration` from the repo

## Describe alternatives you've considered
Open to keeping them if someone shows a unique use case.

### LINKED ISSUE #4596: Using <title> in _document.js's <Head> tag should show a warning to use _app.js instead
# Bug report

Based on #3527

## Describe the bug

Currently, users might add `<title>` in `_document.js` which will lead to unexpected results with `next/head` since `_document.js` is only rendered on the initial pre-render. `_app.js` is rendered on the server and client side, and is rendered as a normal React tree, whereas _document.js is rendered as static HTML.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:
1. Copy the default `_document.js` and add `<title>`:
```
// _document is only rendered on the server side and not on the client side
// Event handlers like onClick can't be added to this file

// ./pages/_document.js
import Document, { Head, Main, NextScript } from 'next/document'

export default class MyDocument extends Document {
  static async getInitialProps(ctx) {
    const initialProps = await Document.getInitialProps(ctx)
    return { ...initialProps }
  }

  render() {
    return (
      <html>
        <Head>
          <title>Hello World</title>
        </Head>
        <body className="custom_class">
          <Main />
          <NextScript />
        </body>
      </html>
    )
  }
}
```

## Expected behavior

Above example should show a warning that `<title>` should only be used in _app.js and other pages. This can be done using `err.sh`, create a link like: `https://err.sh/next.js/no-document-title` and create a file in the `errors` directory: `errors/no-document-title.md`

### PR DESCRIPTION
**Pull request by davscro**
resolves #4498 #4519

### CODE REVIEW

**Review by timneutkens — CHANGES_REQUESTED** [review]
Was reviewing the wrong PR 🙏

**Comment by davscro** [comment]
ok, let me know do I have to correct something. thx

File: examples/with-universal-configuration-build-time/.babelrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "presets": [
+    "next/babel"
+  ],
+  "plugins": [
+    ["transform-define", "./env-config.js"]
```
Comment (timneutkens) [inline examples/with-universal-configuration-build-time/.babelrc]: This won't work with Next 6. It has to be changed to `.babelrc.js` and the `env-config.js` file should be inlined

Comment (davscro) [reply examples/with-universal-configuration-build-time/.babelrc]: committed and fixed with 2557ef1cef77e5571d1f47689606df54f6d9157d

**Pull request closed without merging**